tests/harness.c

```diff
@@ -49,11 +49,11 @@
  */
 #define BOOT_FULL PLAYGROUND_ROOT "/" BOOT_DIRECTORY
 
-#define EFI_START BOOT_FULL "/EFI"
+#define EFI_START BOOT_FULL "/efi"
```

Comment:
Not the original case

Reply:
[This line in v1.5.5](https://github.com/clearlinux/clr-boot-manager/blob/v1.5.5/tests/harness.c#L51). I just restored the original names.